sax.h

```diff
@@ -254,6 +263,10 @@ class MyHandler : public rapidjson::BaseReaderHandler<rapidjson::UTF8<>, MyHandl
     void push_new_key_value(Valuesax& v)//TODO should be reference
     {
       mymap.push_back(std::make_pair(get_key_path(), v));
+      if (get_key_path() == filter) {
```

Comment:
the `get_key_path` is for debug purposes.
it adds '/' to each string to make it human-readable